test/lit/passes/inlining-eh.wast

```diff
@@ -0,0 +1,99 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; RUN: wasm-opt %s --inlining -all -S -o - | filecheck %s
+
+(module
+  ;; ---------------------------------------------------------------------------
+  (import "a" "b" (func $foo (result i32)))
+  (event $event$0 (attr 0) (param i32))
+  (func $callee-with-label
```

Comment:
Why doesn't this function have check lines? If they were manually deleted, they will come back next time someone runs `./auto_update_tests.py`, so it would be better just to keep them in.

Reply:
This is strange. update_lit_checks.py actually doesn't update this. Also `(func $callee` in the same file. Not sure why.

Reply:
Must be a bug in the regex used by the script. I wouldn't worry about it if those functions aren't critical to the test.

Reply:
This indeed can't exist because `$callee-with-label` is inlined! 🙃

Reply:
ohhh, that makes sense. Sorry for the confusion!